<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialPlotter</class>
 <widget class="QMainWindow" name="serialPlotter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Plotter </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cmbSerialPorts">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="currentText">
     <string extracomment="Please Select"/>
    </property>
   </widget>
   <widget class="QLabel" name="labSerialPortStatusDisconnected">
    <property name="geometry">
     <rect>
      <x>798</x>
      <y>4</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>power_off.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Serial Plotter</string>
    </property>
   </widget>
   <widget class="QLabel" name="labSerialPortStatusConected">
    <property name="geometry">
     <rect>
      <x>798</x>
      <y>4</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>electrical_services.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>681</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
